src/alchemtest/amber/access.py

```diff
@@ -75,8 +76,8 @@ def load_simplesolvated():
     """
 
     module_path = dirname(__file__)
-    data = {'charge': glob(join(module_path, 'simplesolvated/charge/*/ti-*.out')),
-            'vdw': glob(join(module_path, 'simplesolvated/vdw/*/ti-*.out'))}
+    data = {'charge': glob(join(module_path, 'simplesolvated/charge/*/ti-*.tar.bz2')),
```

Comment:
I know other parts of the code are using the os.path. Given that you are using pathlib anyway here, we could have all the new code using the pathlib.

Reply:
Nice hint, I got rid of both glob and os.patn, now AMBER is totally Path-centric XD